tutorials/wpa-enterprise.rst

```diff
@@ -0,0 +1,373 @@
+How to Set Up WPA Enterprise (EAP-TTLS-PAP) authentication
+==========================================================
+
+In this tutorial, we will guide you set up WPA Enterprise (EAP-TTLS-PAP)
+authentication for WiFi networks using OpenWISP. The RADIUS capabilities
+of OpenWISP provides integration with FreeRADIUS to allow users to
+authenticate with their django user accounts. Users can either be created
+manually via the admin interface, generated or imported from CSV.
+
+You can try this feature on our :doc:`OpenWISP Demo System <./demo>`!
```

Comment:
I think this line is misleading. It may be misinterpreted by users who may think the tutorial is mainly aimed at Demo which is not the case. Since Demo is mentioned below, I think we can avoid it here.